about-peer-review/policies-guidelines.md

```diff
@@ -2,31 +2,25 @@
 
 ## Review Process Guidelines
 
-- Packages are reviewed for quality, fit, documentation, clarity and the review process
-  is quite similar to a manuscript review (see our [packaging guide](../authoring/overview)
-  and [reviewing guide](../peer-review-guides/reviewer-guide) for more details). Unlike a
-  manuscript review, this process will be an ongoing conversation.
-- Once all major issues and questions, and those addressable with reasonable effort, are
-  resolved, the editor assigned to a package will make a decision (accept, hold, or
-  reject). Rejections are usually done early (before the review process begins, see the
-  aims and scope section). In rare cases a package may also not be on-boarded after
-  review & revision. It is ultimately editor’s decision on whether or not to reject the
-  package based on how the reviews are addressed.
-- Communication between authors, reviewers and editors will first and foremost take
-  place on GitHub, although you can choose to contact the editor by email. When
-  submitting a package, please make sure your GitHub notification settings make it
-  unlikely you will miss a comment.
-- The author can choose to have their submission put on hold (editor applies the holding
-  label). The holding status will be revisited every 3 months, and after one year the
-  issue will be closed.
-- If the author hasn’t requested a holding label, but is simply not responding, we
-  should close the issue within one month after the last contact intent. This intent
-  will include a comment tagging the author, but also an email using the email address
-  listed in the DESCRIPTION of the package which is one of the rare cases where the
-  editor will try to contact the author by email.
-- If a submission is closed and the author wishes to re-submit, they’ll have to start a
-  new submission. If the package is still in scope, the author will have to respond to
-  the initial reviews before the editor starts looking for new reviewers.
+pyOpenSci packages are reviewed for quality, fit, scope, 
+documentation and usability. The review process
+is similar to a manuscript review, however it has a stronger 
+focus on Python packaging best practices. 
+
+Unlike a manuscript review, our peer review process is be an ongoing conversation. Once all major issues and questions are addressed, the review editor package will make a decision to accept, hold, or reject the package. 
+
+Rejections are usually done early in the process, before the review process begins. In rare cases a package may also not be on-boarded into the pyOpenSci ecosystem after review & revision. 
+
+It is ultimately editor’s decision on whether or not to reject the package based on how the reviews are addressed.
+
+## Review communication approach 
+
+Communication between authors, reviewers and editors takes 
+place on GitHub. You can, however choose to contact the editor by email if needed. 
```

Comment:
this is a great question @arianesasso in some cases I think there is a place for email. Many people miss github "pings" for instance i might have hundreds in a day of emails with github's name on them. Ideally i'd see it but sometimes i dont. So i think it's ok to email people too. but i totally hear your point and like that we can start by pinging on github. 

oh another point -- in some cases maybe someone wants to review but they'd like some help or support. it might be more comfortable for them to acknowledge that in an email as well vs in a  public issue. So i think there is a place for both! but again excellent insight here :)